studio/LonaStudio/Workspace/TextStyleEditorViewController.swift

```diff
@@ -36,27 +36,108 @@ class TextStyleEditorViewController: NSViewController {
     public var textStyles: [CSTextStyle] = [] { didSet { update() } }
     public var onInspectTextStyle: ((CSTextStyle?) -> Void)?
 
-    // Change colors and specify the selection simultaneously
+    // Change text styles and specify the selection simultaneously
     public var onChangeTextStyles: ((String, [CSTextStyle], CSTextStyle?) -> Void)?
 
     // MARK: Private
 
     private let textStyleBrowser = TextStyleBrowser()
+    private lazy var addTextStyleSheet = AddTextStyleSheet()
+    private lazy var addTextStyleViewController: NSViewController = {
+        return NSViewController(view: addTextStyleSheet)
+    }()
 
     private func setUpViews() {
         self.view = textStyleBrowser
 
-        textStyleBrowser.onSelectTextStyle = { color in
-            self.onInspectTextStyle?(color)
+        textStyleBrowser.onSelectTextStyle = { textStyle in
+            self.onInspectTextStyle?(textStyle)
         }
 
-        textStyleBrowser.onClickAddTextStyle = {}
+        textStyleBrowser.onClickAddTextStyle = {
+            var textStyle = CSTextStyle(id: "", name: "", fontName: nil, fontFamily: nil, fontWeight: nil, fontSize: nil, lineHeight: nil, letterSpacing: nil, color: nil, comment: nil)
+            var pristineIdText: Bool = true
+
+            self.addTextStyleSheet.idText = textStyle.id
```

Comment:
I think we might need to assign default values to all the other fields too? I'm not 100% sure, but it seems like the color value at least isn't getting reset after I make a new color and open the sheet again.